package.json

```diff
@@ -2,20 +2,26 @@
   "name": "jest-preset-angular",
   "version": "8.0.0",
   "description": "Jest preset configuration for Angular projects",
-  "main": "index.js",
+  "main": "build/index.js",
   "repository": "git@github.com:thymikee/jest-preset-angular.git",
   "homepage": "https://github.com/thymikee/jest-preset-angular",
   "author": "Michał Pierzchała <thymikee@gmail.com>",
   "license": "MIT",
   "dependencies": {
     "jest-environment-jsdom-fifteen": "^1.0.0",
-    "pretty-format": "^24.0.0",
-    "ts-jest": "^24.0.0"
```

Comment:
since `ts-jest` is no longer installed automatically by this preset anymore, we should add this to the breaking part of `CHANGELOG`

Reply:
Done!